backend/src/bundles/connections/connection.controller.ts

```diff
@@ -0,0 +1,132 @@
+import { type OAuthService, HttpCode } from '~/bundles/oauth/oauth.js';
+import { type UserAuthResponseDto } from '~/bundles/users/users.js';
+import {
+    type ApiHandlerOptions,
+    type ApiHandlerResponse,
+    ApiHandlerResponseType,
+    BaseController,
+} from '~/common/controller/controller.js';
+import { ApiPath } from '~/common/enums/enums.js';
+import { type Logger } from '~/common/logger/logger.js';
+
+import { ConnectionsPath } from './enums/enums.js';
+
+/**
+ * @swagger
+ * components:
+ *   schemas:
+ *     Error:
+ *       type: object
+ *       properties:
+ *         errorType:
+ *           type: string
+ *           enum:
+ *              - COMMON
+ *              - VALIDATION
+ *         message:
+ *           type: string
+ *
+ */
+
+/**
+ * @swagger
+ * components:
+ *    schemas:
+ *      OAuth:
+ *        type: object
+ *        properties:
+ *          id:
+ *            type: number
+ *            format: number
+ *            minimum: 1
+ *          userId:
+ *            type: number
+ *            format: number
+ *            minimum: 1
+ *          tokenType:
+ *            type: string
+ *          expiresAt:
+ *            type: number
+ *            format: number
+ *            minimum: 0
+ *          accessToken:
+ *            type: string
+ *          refreshToken:
+ *            type: string
+ *          scope:
+ *            type: string
+ *            nullable: true
+ *          provider:
+ *            type: string
+ *            enum:
+ *              - strava
+ *              - google-fit
```

Comment:
you could create something like toPublicObject() in entity to map into response, there is no reason to send tokens, tokenType, expiresAt